main.tf

```diff
@@ -0,0 +1,80 @@
+/**
+ * # Jenkins Infrastructure Code for SDS platform
+ *
+ *
+ *
+ */
+
+
+module "tags" {
+  source      = "git::https://github.com/hmcts/terraform-module-common-tags.git?ref=master"
+  environment = lower(var.environment)
+  product     = var.product
+  builtFrom   = var.builtFrom
+}
+
+
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.product}-${var.environment}-rg"
+  location = var.location
+  tags     = module.tags.common_tags
+}
+
+resource "azurerm_managed_disk" "disk" {
+  name                 = "${var.product}-${var.environment}-disk"
+  location             = var.location
+  resource_group_name  = azurerm_resource_group.rg.name
+  storage_account_type = "Premium_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = "1024"
+  tags                 = module.tags.common_tags
+
+}
+
+resource "azurerm_user_assigned_identity" "usermi" {
+  resource_group_name = azurerm_resource_group.rg.name
```

Comment:
this should go in the specific resource group for it so that the kubernetes cluster has the right RBAC setup to manage the identity:

https://github.com/hmcts/aks-module-kubernetes/blob/master/12-kubernetes-cluster.tf#L10

Reply:
do i create the RG as part of this code